### PR TITLE
Bottom left side radio selector

### PR DESCRIPTION
**Pull request by MangeFre**
[Ticket](https://trello.com/c/28I28LK6)

Hoping this works well enough! Using "fake data" right now, but could easily implement some event dispatcher that updates the store whenever a data set is selected.

### CODE REVIEW

**Comment by MangeFre** [comment]
Keep this on hold as I want to merge in Ritesh latest PR

File: src/components/DatasetSelector.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+	// const { options } = props;
+
+    const [selectedOption, setSelectedOptions] = useState(
+		options[0] ? options[0] : null,
+	);
+
+	const handleSelection = (e, index) => {
+		console.log(e);
+		console.log(index);
```
Comment (mcdevittbass) [inline src/components/DatasetSelector.js:22]: Unless there is a reason to leave them in, can you take our your console logs? Also the comment in line 14.

File: src/App.css [hunk 2]
```diff
@@ -37,11 +47,126 @@
...
+  padding: 0px 0px 0px 1rem;
+}
+
+#data-selector label {
+  font: normal normal normal 0.8125rem/1.0625rem futura-pt;
+  letter-spacing: 0px;
+  color: #1C752A;
+  opacity: 1;
+  font-weight: 400;
+}
```
Comment (mcdevittbass) [inline src/App.css:104]: Screen Shot 2021-05-19 at 10 31 12 PM

Maybe it's related to the above component not being up to design specs yet, but can your component be a little bit wider and with less margin at the bottom (under the last list item)? Also maybe we can give the whole left column a gray background like this so there is not white under it?

Comment (mcdevittbass) [reply src/App.css:104]: Also bolder and more centered for the title? [https://xd.adobe.com/view/915910be-fb03-46d4-a5df-e8e1109c8e31-c40a/screen/5aa15b4b-e156-47f5-938a-a9168feb8866/](url)

**Review by mcdevittbass — CHANGES_REQUESTED** [review]
@MangeFre your code looks great! Really good job! Just had a couple of minor tweaks to request.

File: src/App.css (hunk 2 above)
Comment (MangeFre) [reply src/App.css]: Hey, so I've made a quite a lot of directory structure changes (the thing Ritesh and I spoke about during the meetings) and also fixed the CSS issues you pointed out. Let me know what you think.

Small points: 
1. The bottom left container looks better now in terms of size, but it is dependent on a hardcoded height of the "ZoomToBoundsMenu" (see App.css, line 37). I don't have a better solution atm :/

2. The width of the selector is too big, but it is based on the width of the whole column, which is not in line with the size of the prototype. I think that might be a different PR so I want this merged in first so it doesn't keep growing in all directions.

Hope this works better! Let men know if there are more things to tweak.

**Review by maharit108 — APPROVED** [review]

**Review by mcdevittbass — APPROVED** [review]
lgtm!

**Pull request merged**